app/uk/gov/hmrc/eusubsidycompliancefrontend/views/BusinessEntityEoriPage.scala.html

```diff
@@ -52,6 +54,8 @@
       )
   }
 
+@if(appConfig.xiEoriAdding) {
```

Comment:
this is the wrong way around. We want the old behaviour by default and only see the new functionality when the flag is true

Reply:
updated